Lesson7.ipynb

```diff
@@ -29,6 +33,551 @@
+      "['12', 'Scrambled Legs', 'TRUE', '12-5', '20592', '19092', '20154', '17551', '10438', '19226', '14786', '24248', '24577', '21135', '21846', '14786', '16138', '343', '21853', '22762', '20615', '4110', '6299', '3435', '2078', '5960', '13194', '6922', '7970', '7444', '8126', '11797']\n",
+      "['13', 'Smarty Pints', 'FALSE', '13-1', '10003', '11253', '24000', '6650', '13027', '9134', '6674', '19353', '4179', '6765', '9252', '4051', '5984', '1722', '5879', '4299', '4317', '4080', '3240', '9055', '2312', '3967', '2834', '6517', '7766', '3318', '5807', '5114']\n",
+      "['13', 'Smarty Pints', 'FALSE', '13-2', '10166', '7333', '13278', '7034', '12428', '4152', '4762', '20641', '17698', '23264', '24754', '21951', '19814', '10297', '', '', '', '', '', '', '', '', '', '', '', '', '', '']\n",
+      "['13', 'Smarty Pints', 'FALSE', '13-3', '9311', '4711', '11085', '4426', '1635', '1495', '16395', '18993', '9371', '13832', '11324', '10242', '14231', '5772', '10262', '12526', '', '', '', '', '', '', '', '', '', '', '', '']\n",
+      "['13', 'Smarty Pints', 'TRUE', '13-4', '13880', '17436', '18092', '18089', '19117', '12453', '7883', '7795', '16531', '6200', '8882', '7822', '8253', '7731', '18922', '21463', '14563', '13433', '10143', '5678', '4527', '22371', '16636', '', '', '', '', '']\n",
+      "['13', 'Smarty Pints', 'FALSE', '13-5', '6229', '8772', '6815', '9598', '5169', '8327', '11633', '5098', '7396', '3013', '9215', '8513', '4402', '10973', '4923', '5118', '8616', '4693', '16140', '6760', '7153', '3818', '7159', '4197', '12842', '8780', '8294', '']\n",
+      "['14', '', 'FALSE', '14-1', '7665', '13442', '15127', '10702', '11158', '16580', '12752', '7429', '14176', '4692', '16287', '10416', '14343', '13534', '3823', '12855', '11594', '13175', '4116', '20587', '4248', '11269', '5594', '17626', '10839', '13391', '23290', '10965']\n",
+      "['14', '', 'FALSE', '14-2', '', '6076', '8365', '13014', '16050', '7265', '8690', '12492', '10100', '8352', '14117', '14317', '15684', '4678', '17677', '15898', '3564', '16544', '12659', '10634', '2065', '18566', '6632', '9739', '16554', '10234', '1055', '8066']\n",
+      "['14', '', 'FALSE', '14-3', '11328', '4887', '5037', '13627', '6243', '14736', '5100', '11238', '4886', '3964', '5764', '10180', '3259', '6302', '4537', '7275', '78', '2969', '1989', '11312', '5910', '', '', '', '', '', '', '']\n",
+      "['14', '', 'FALSE', '14-4', '942', '7664', '2600', '10054', '', '7397', '', '', '', '15585', '15000', '11000', '', '', '', '', '', '', '', '', '', '', '', '', '', '', '', '']\n",
+      "['14', '', 'FALSE', '14-5', '3175', '3853', '4174', '6633', '1553', '1675', '3265', '4693', '2987', '4383', '2640', '922', '2791', '4261', '4999', '3032', '4678', '3701', '5082', '5665', '6311', '3983', '2954', '3960', '3343', '3862', '6578', '3139']\n"
+     ]
```

Comment:
We usually clear all the outputs before submitting a lesson:
Menu > Edit > Clear all Outputs